packages/web/lib/networking/queries/useGetLibraryItemsQuery.tsx

```diff
@@ -192,6 +192,14 @@ export function useGetLibraryItemsQuery({
     }
   }
 
+  const getIndexOf = (page: LibraryItems, item: LibraryItem) => {
+    for (let i = 0; i < page.edges.length; i++) {
```

Comment:
I think here we can use `findIndex` and it might be a little cleaner:

```
> ['a', 'aaa'].findIndex(i => i.length > 1)
1
```

Reply:
@jacksonh We have updated this function.